docs/CHANGELOG.rst

```diff
@@ -58,6 +58,8 @@ Changed
 Fixed
 -----
 * Fix ``consensus_subreads`` input option in Subread process
+* Fix memory limit in vc_chemut process
```

Comment:
Please write more informative log entries. We have started to use the change log for reporting new features to the end users. Not directly of course, but I am certain @jinlajf and @rafecooks would appreciate a more verbose list of changes. I would suggest to write something like "The chemical mutagenesis analysis crashed with the out-of-box 16 GB limit for some samples <is there a common denominator for the samples where this crashed, like for Human samples or for samples larger then...>. Therefore we have increased the memory limit to 32 GB".